salt/modules/suse_apache.py

```diff
@@ -23,7 +23,7 @@ def __virtual__():
     '''
     Only load the module if apache is installed.
     '''
-    if salt.utils.path.which('apache2ctl') and __grains__['os_family'] == 'SUSE':
+    if salt.utils.path.which('apache2ctl') and __grains__['os_family'] == 'Suse':
```

Comment:
could we just ignore case, by adding lower() call there? or something similar?

Also can we get a test added here. We do run branch tests on suse 15, so we can ensure it still loads there with a test.

Reply:
Sure, something like 
`if salt.utils.path.which('apache2ctl') and __grains__['os_family'].casefold() == 'suse':`
would also work. In this case I suppose changing all grains matching code in the module to avoid problems with case changes in the future and provide uniformity.
Should I test this and adjust the commit?

Reply:
yep that will work. let me know if you have any questions about tests.